bootcd/rpms/genesis_scripts/genesis_scripts.spec

```diff
@@ -11,6 +11,9 @@ Source2:        src/sysconfig-init.diff
 Source3:        src/tty.conf.override
 Source4:        src/genesis-bootloader
 Source5:        src/login-shell
+Source6:        src/init.d-genesis
```

Comment:
should I also clean up line 9 by moving and renaming the existing `init.d-network-prep`?